### PR TITLE
feat(ui): add tooltip component

### PR DESCRIPTION
**Pull request by dudiq**

### CODE REVIEW

File: packages/ui/src/kit/VHover/VHover.vue [hunk 1]
```diff
@@ -0,0 +1,24 @@
+<template>
+  <div
+    v-bind="$attrs"
+    @mouseover="isHover = true"
```
Comment (lamartire) [inline packages/ui/src/kit/VHover/VHover.vue]: Move that to methods

File: packages/ui/src/kit/VTooltip/VTooltip.theme-default.scss [hunk 2]
```diff
@@ -0,0 +1,85 @@
...
+  left: 12px;
+}
+.v-tooltip.position-bottom-left .v-tooltip-container {
+  left: 50%;
+  margin-left: -16px;
+  margin-bottom: 9px;
+  bottom: 0
+}
+
+.v-tooltip.position-bottom-right .v-tooltip-triangle {
```
Comment (lamartire) [inline packages/ui/src/kit/VTooltip/VTooltip.theme-default.scss]: Better do that with `:after`

File: packages/ui/src/kit/VTooltip/VTooltip.theme-default.scss [hunk 3]
```diff
@@ -0,0 +1,85 @@
...
+  margin-top: 9px;
+}
+
+.v-tooltip.position-top-right .v-tooltip-triangle {
+  top: -5px;
+  right: 12px;
+}
+.v-tooltip.position-top-right .v-tooltip-container {
+  right: 50%;
+  margin-right: -16px;
```
Comment (lamartire) [inline packages/ui/src/kit/VTooltip/VTooltip.theme-default.scss]: Are you sure that it is ok to combine `margin-right` with `right` in `abosolute` positioned element? Use `calc()` function or `transform`.

File: packages/ui/src/kit/VTooltip/VTooltip.vue [hunk 4]
```diff
@@ -0,0 +1,51 @@
...
+    show: {
+      type: Boolean,
+      default: false,
+    },
+    position: {
+      type: String,
+      default: 'top-left',
+      validator(value) {
+        return (
+          ['top-left', 'top-right', 'bottom-left', 'bottom-right'].indexOf(
```
Comment (lamartire) [inline packages/ui/src/kit/VTooltip/VTooltip.vue]: I think tooltip should have `top`, `left` etc position variants, not current. It is common behavior for tooltip.

File: packages/ui/src/kit/VTooltip/VTooltip.vue [hunk 5]
```diff
@@ -0,0 +1,51 @@
...
+            value,
+          ) !== -1
```
Comment (lamartire) [inline packages/ui/src/kit/VTooltip/VTooltip.vue]: `includes`

File: packages/ui/src/kit/VTooltip/VTooltip.theme-default.scss (hunk 3 above)
Comment (dudiq) [reply packages/ui/src/kit/VTooltip/VTooltip.theme-default.scss]: i'm sure, it's oldschool story

File: packages/ui/src/kit/VTooltip/VTooltip.vue (hunk 4 above)
Comment (dudiq) [reply packages/ui/src/kit/VTooltip/VTooltip.vue]: don't think soo, if we need center positions in top or bottom - it will be not a certain story =\

Comment (dudiq) [reply packages/ui/src/kit/VTooltip/VTooltip.vue]: just like `top-center` or `bottom-center`

**Pull request merged**